src/main/java/ua/mykolamurza/chatullo/handler/ChatHandler.java

```diff
@@ -62,8 +104,8 @@ private void writeToLocalChat(AsyncChatEvent event, Player player, String messag
     }
 
     private boolean isPlayerHearLocalChat(Player player, Player viewerPlayer) {
-        return viewerPlayer.getWorld().equals(player.getWorld())
-                || viewerPlayer.getLocation().distance(player.getLocation()) <= getLocalChatDistance();
+        return !(!viewerPlayer.getWorld().equals(player.getWorld())
```

Comment:
Unclear. Had this line contained a bug? 
Looks like just an inverted statement.